### ISSUE TITLE
Use open data instead?

### ISSUE THREAD

**Issue by grischard** [eLvErDe/find-yellow-post-box-in-luxembourg-city#1]
Would https://data.public.lu/en/datasets/adresses-et-heures-de-releve-des-boites-aux-lettres-post-luxembourg/ do the trick?

**Comment by eLvErDe** [comment]
Hi,

Yeah that's basically what I extracted by scrapping the website. Proper JSON API would be a lot better though ;)

Adam.

**Comment by grischard** [comment]
Indeed. Something like https://github.com/grischard/postcode-centroid can be done to fetch the latest from data.public.lu, but it would almost be worth it to have a package on pypi...
